fury/actor.py

```diff
@@ -2383,6 +2383,17 @@ def billboard(centers, colors=(0, 1, 0), scales=1, vs_dec=None, vs_impl=None,
     return bb_actor
 
 
+def billboard_gs(centers, colors, scales=(1, 1, 1)):
```

Comment:
@filipinascimento I think it would be better to have both functionalities. We have seen how some things break when there's a major release of VTK, especially things related to shaders. If that happens in the future, we'll just change the default parameter to the original functionality while we fix the shaders.

Regarding the properties, the purpose of changing the dot actor in #584 was precisely this, using a new dot actor as the basis for a GS version of the billboard. In that case, the colors are handled by the polydata in the dot actor.